src/VS.Web.CG.EFCore/NavigationMetadata.cs

```diff
@@ -21,8 +21,9 @@ public NavigationMetadata(INavigation navigation, Type dbContextType)
             Contract.Assert(navigation.IsDependentToPrincipal());
 
             AssociationPropertyName = navigation.Name;
-
+#pragma warning disable EF1001 // Internal EF Core API usage.
             var otherEntityType = navigation.ForeignKey.ResolveOtherEntityType(navigation.DeclaringEntityType);
```

Comment:
@AndriySvyryd Is there a good alternative to using this internal API?

Reply:
No, I'll make it public in the next preview.

Reply:
Actually, in this case `navigation.GetTargetType()` can be used instead.